packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -0,0 +1,213 @@
+  ${display('inline')}
+  :host .control {
+    display: flex;
+  }
+  :host .date-view {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: ${spacingHorizontalM};
+    width: 248px;
+    height: 285px;
+    outline: solid black;
+    color: ${colorNeutralForeground1};
+    font: ${fontWeightRegular} ${fontSizeBase300} / ${lineHeightBase300} ${fontFamilyBase};
+    border-radius: ${borderRadiusMedium};
+  }
+  :host .right-panel {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: ${spacingHorizontalM};
+    outline: solid lightblue;
+    border-radius: ${borderRadiusMedium};
+    font: ${fontWeightRegular} ${fontSizeBase300} / ${lineHeightBase300} ${fontFamilyBase};
+  }
+  :host .right-cell-outer:not(.right-panel-today):hover {
+    background: ${colorBrandBackgroundInvertedHover};
+    color: ${colorNeutralForeground1Static};
+  }
+  :host .day:not(.today):active {
+    background: ${colorBrandBackgroundInvertedSelected};
+    color: ${colorNeutralForeground1Static};
+  }
+  :host .title,
+  .right-panel-title {
+    position: inline;
+    padding: ${spacingHorizontalM};
+    font: ${fontWeightBold} ${fontSizeBase200} / ${lineHeightBase200} ${fontFamilyBase};
+    color: ${colorNeutralForeground1};
+  }
+  :host .right-panel-title {
+    cursor: pointer;
+  }
+  :host .right-panel-row {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    border-bottom: none;
+    justify-items: center;
+    padding: 2px 0px;
+  }
+  :host .right-cell-outer,
+  .right-cell {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    white-space: normal;
+    height: 44px;
+    width: 44px;
+    border-radius: ${borderRadiusMedium};
+  }
+  :host .week-days,
+  .week {
+    color: ${colorNeutralForeground3};
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    border-left: var(--cell-border, none);
+    border-bottom: none;
+    padding: 0;
+  }
+  :host .interact .week {
+    grid-gap: calc(var(--design-unit) * 1px);
+    margin-top: calc(var(--design-unit) * 1px);
+  }
+  :host .day,
+  .week-day {
+    border-bottom: var(--cell-border);
+    border-right: var(--cell-border);
+    padding: var(--cell-padding);
+    border-radius: 4px;
+  }
+  :host .week-day {
+    text-align: center;
+    border-radius: 0;
+    border-top: var(--cell-border);
+    color: ${colorNeutralForeground3};
+  }
+  :host .day {
+    box-sizing: border-box;
+    vertical-align: top;
+    outline-offset: -1px;
+    line-height: 32px;
+    white-space: normal;
+    width: 32px;
+  }
+  :host .interact .day,
+  .right-cell {
+    background: var(--neutral-fill-rest);
+    cursor: pointer;
+  }
+  :host .day.inactive {
+    color: ${colorNeutralForegroundDisabled};
+    background: ${colorTransparentBackground};
+  }
+  :host .day.disabled {
+    background: var(--disabled-day-background);
```

Comment:
Oops, it's messy here because I haven't really worked on the styling, so a lot of this was copied over from FAST. They're not real variables! I'm waiting to get to styling until after I finish functionalities, since class names, behavior, etc. might change. I'll update these variables to Fluent tokens then.

Reply:
fixed